crates/bmp-pkt/src/wire/deserializer/mod.rs

```diff
@@ -58,6 +58,26 @@ impl BmpParsingContext {
     pub fn new(map: HashMap<PeerKey, BgpParsingContext>) -> Self {
         Self(map)
     }
+
+    fn peer_count(&self) -> usize {
```

Comment:
these functions need to be `pub` otherwise compilation is failing. Any private functions that is not used will fail compilation, to avoid dead code.

Reply:
absolutely, my bad. i'm pretty sure i ran clippy and it did not catch them strangely

Reply:
It was caught by the compiler, https://github.com/NetGauze/NetGauze/actions/runs/11134771994/job/30943604901#step:5:387

Clippy complements the compiler for more opinionated stuff but it doesn't replace it, I had it configured here: https://github.com/NetGauze/NetGauze/blob/main/.cargo/config.toml#L11